.github/workflows/ci_emmocheck.yml

```
@@ -0,0 +1,29 @@
+name: CI tests
+on: [push]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install EMMO-python
+      run: |
+        pip install EMMO
```

No need for the pipe (also below), or are you future-proofing for more lines? :)
```suggestion
      run: pip install EMMO
```